org.eclipse.xtext.ui.shared/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,7 @@ Bundle-Version: 2.13.0.qualifier
 Bundle-Localization: plugin
 Bundle-Activator: org.eclipse.xtext.ui.shared.internal.Activator
 Bundle-Vendor: Eclipse Xtext
-Require-Bundle: org.eclipse.xtext.ui;resolution:=optional;x-installation:=greedy,
+Require-Bundle: org.eclipse.xtext.ui;bundle-version="[0.0.0,2.14.0)";resolution:=optional;x-installation:=greedy,

```

Comment:
My gut feeling is, that we should always have exact matches between the xtext bundles. It's unlikely that xtext.ui will ever reliably work with another version of xtext or xtext.ui.shared.